src/commands/convert-openapi-document.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright © 2023, Oracle and/or its affiliates.
+ * This software is licensed to you under the Universal Permissive License (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+import * as vscode from 'vscode';
+
+import * as _fs from 'fs';
+
+import { bindNodeCallback, catchError, filter, firstValueFrom, from, map, switchMap, tap, throwError } from 'rxjs';
```

Comment:
We discussed this before. Do you plan to move away from rxjs?

Reply:
This is not for API calls, but mostly UI works.